Agent/Public/NRLogger.h

```diff
@@ -132,19 +147,11 @@ withMessage:(NSString *)message;
  */
 + (void)setLogTargets:(unsigned int)targets;
 
-/*
-Configure the New Relic logging API.
+// For internal use only. Do not use.
++ (void)setLogIngestKey:(NSString*)key;
```

Comment:
Consider using an internal header here, and adding a category, so we don't have to directly expose our internal APIs.